src/ui/KonvergoWindow.cpp

```diff
@@ -253,6 +271,27 @@ void KonvergoWindow::updateFullscreenState(bool saveGeo)
 }
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////
+void KonvergoWindow::updateAlwaysOnTopState()
+{
+  QLOG_DEBUG() << "Changing always-on-top state";
+  Qt::WindowFlags forceOnTopFlags = Qt::WindowStaysOnTopHint;
+#ifdef Q_OS_LINUX
+  forceOnTopFlags = forceOnTopFlags | Qt::X11BypassWindowManagerHint;
```

Comment:
This is a X11 specific thing right? In that case you should probably use Q_WS_X11 otherwise it happens on raspberry pi as well.